File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex24.java
//Importando Scanner
import java.util.Scanner;


public class Ex24 {
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double dinheiro, marco, dolar, libra;
	
	
	System.out.println("Conversão de dinheiro\n\n");
	
	//Entrada	
	System.out.print("Insira a quantidade de dinheiro em reais: ");
	dinheiro = ent.nextDouble();
	
	//Processamento
	dolar = dinheiro * 1.8;
	marco = dinheiro * 2;
	libra = dinheiro * 1.57;
	
	//Saída
	System.out.printf("Dólares: %.2f\n", dolar);
	System.out.printf("Marco alemão: %.2f\n", marco);
	System.out.printf("Libras esterlinas: %.2f", libra);
	
	
		
	}

}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex24.c
#include <stdio.h>

main()
{
    //vars
    float dinheiro, dolares, marco, libras;

    printf("\t\t\tConversao de dinheiro\n\n");

    //Entrada
    printf("Insira a quantia em reais: ");
    scanf("%f",&dinheiro);

    //Processamento
    dolares = dinheiro * 1.8;
    marco = dinheiro * 2;
    libras = dinheiro * 1.57;

    //Saída
    printf("Conversao para:\n");
    printf("-dolares: %.2f\n", dolares);
    printf("-marco alemao %.2f\n", marco);
    printf("-libras esterlinas: %.2f\n", libras);

}

